Sources/Knit/Resolver.swift

```diff
@@ -13,4 +13,50 @@ public protocol Resolver: AnyObject {
 
     func unsafeResolver(file: StaticString, function: StaticString, line: UInt) -> Swinject.Resolver
 
+    init(_swinjectContainer: Swinject.Container)
```

Comment:
I'm surprised this is part of the protocol interface

Reply:
It's needed so the Container can instantiate the Resolver based on the type